app/notification_push.go

```diff
@@ -273,7 +273,12 @@ func (a *App) StopPushNotificationsHubWorkers() {
 func (a *App) sendToPushProxy(msg model.PushNotification, session *model.Session) {
 	msg.ServerId = a.DiagnosticId()
 
-	request, _ := http.NewRequest("POST", strings.TrimRight(*a.Config().EmailSettings.PushNotificationServer, "/")+model.API_URL_SUFFIX_V1+"/send_push", strings.NewReader(msg.ToJson()))
+	request, err := http.NewRequest("POST", strings.TrimRight(*a.Config().EmailSettings.PushNotificationServer, "/")+model.API_URL_SUFFIX_V1+"/send_push", strings.NewReader(msg.ToJson()))
+	if err != nil {
+		mlog.Error(fmt.Sprintf("Error sending to push proxy: UserId=%v SessionId=%v message=%v",
```

Comment:
0/5, but `UserId` is duplicated twice in this log, given the `"user_id"` structured parameter. Worth converting them to structured parameters across the board? (including `mlog.Err(err)` to get an `err=...` field?)

Reply:
In this PR, copied from adjacent code, but it seemed redundant to me too